kbase-extension/static/kbase/js/widgets/function_output/kbaseGenomeAnnotationViewer.js

```diff
@@ -344,341 +350,397 @@ define (
 
-          var $target = $.jqElem('div').append($self.loaderElem());
+            var $target = $.jqElem('div').append($self.loaderElem());
 
-          var $selector = $.jqElem('select')
-            .on('change', function() {
-              $self.showContent($selector.val(), $target);
+            var $selector = $.jqElem('select').on('change', function () {
+                $self.showContent($selector.val(), $target);
+            });
+            for (type in Object.keys(this.summary.annotation.feature_type_counts).sort(
+                this.sortCaseInsensitively
+            )) {
+                $selector
+                    .append(
+                        $.jqElem('option')
+                            .attr('value', type)
+                            .attr('selected', type == 'gene' ? 'selected' : undefined)
+                            .append(type)
+                    )
+                    .addClass('form-control input-sm')
+                    .css({ width: 'auto', display: 'inline' });
             }
-          );
-          for (type in Object.keys(this.summary.annotation.feature_type_counts).sort(this.sortCaseInsensitively)) {
-            $selector.append(
-              $.jqElem('option')
-                .attr('value', type)
-                .attr('selected', type == 'gene' ? 'selected' : undefined)
-                .append(type)
-            ).addClass('form-control input-sm').css({'width':'auto', 'display':'inline'});
-          }
-
-
-          var $container = $.jqElem('div')
-            .append(
-              $.jqElem('div')
-                .css('padding-top','10px')
-                .append("Showing Feature Type:  ")
-                .append($selector)
-            )
-            .append($target);
 
-          $tab.append($container);
-
-          $tab.hasContent = true;
+            var $container = $.jqElem('div')
+                .append(
+                    $.jqElem('div')
+                        .css('padding-top', '10px')
+                        .append('Showing Feature Type:  ')
+                        .append($selector)
+                )
+                .append($target);
 
-          this.showContent('gene', $target);
-          return;
-        },
+            $tab.append($container);
 
+            $tab.hasContent = true;
 
-        setDataTableStyles: function($table) {
-          return $table
-                    .addClass('table table-striped table-bordered table-hover')
-                    .css({'width':'100%', 'border':'1px solid #ddd', 'margin-left': 'auto', 'margin-right':'auto'});
+            this.showContent('gene', $target);
+            return;
         },
 
+        setDataTableStyles: function ($table) {
+            return $table.addClass('table table-striped table-bordered table-hover').css({
+                width: '100%',
+                border: '1px solid #ddd',
+                'margin-left': 'auto',
+                'margin-right': 'auto',
+            });
+        },
```

Comment:
This is the only place in this file where prettier has introduced a change to the abstract syntax tree of the file. From a code perspective, there is no difference between these lines and lines 395-399 in the old file.